README.md

```diff
@@ -72,6 +68,17 @@ Installation using [bower](http://bower.io):
 CLDR does NOT use Github's issue tracking system to track bugs.  If you find an error in
 the data contained here, please file a new ticket at [Unicode Jira](https://unicode-org.atlassian.net/projects/CLDR/issues)
 
+### Licenses
+
+- Usage of CLDR data and software is governed by the [Unicode Terms of Use](http://www.unicode.org/copyright.html)
+a copy of which is included as [LICENSE](./LICENSE).
```

Comment:
It looks from this PR as if `./LICENSE` actually contains a copy of http://www.unicode.org/license.html, not copyright.html?

Reply:
copyright.html has a link to license, and explains about the copyright on the data as well. So I'd rather link to copyright.html

Reply:
in the license it says "Distributed under the Terms of Use in https://www.unicode.org/copyright.html. "